demo/denas/bert/DENAS_BERT_DEMO.ipynb

```diff
@@ -823,11 +823,11 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.12"
+   "version": "3.9.12 (main, Aug 29 2022, 00:54:58) \n[GCC 11.2.0]"
   },
   "vscode": {
```

Comment:
what's this? @tianyil1

Reply:
This is automaticall updated with the local editor, which does not influence the result. @Jian-Zhang